ibm/service/power/resource_ibm_pi_instance_console_language.go

```diff
@@ -32,12 +30,12 @@ func ResourceIBMPIInstanceConsoleLanguage() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			// Required Attributes
-			helpers.PICloudInstanceId: {
+			Arg_CloudInstanceID: {
 				Type:        schema.TypeString,
```

Comment:
Sort all stanzas alphabetically and match their descriptions to what is on the documentation.